src/reducers/index.js

```diff
@@ -11,10 +11,16 @@ const initialState = {
 // need to do with the count in each case?
 export default (state = initialState, action) => {
   switch (action.type) {
-    case INCREMENT:
-    // Fill in the body of this case
+    case INCREMENT: 
+    return {
+      ...state,
+       count: state.count + 1
+      }
     case DECREMENT:
-    // Fill in the body of this case
+    return {
+      ...state,
+       count: state.count - 1
```

Comment:
awesome, yeah. the action in this case doesn't have a payload, we're hard-coding in the `1` to either increment or decrement